src/main/java/com/coreos/jetcd/ClientBuilder.java

```diff
@@ -145,6 +148,20 @@ public ClientBuilder setLazyInitialization(boolean lazyInitialization) {
     return this;
   }
 
+  public SslContext getSslContext() {
+    return sslContext;
+  }
+
+  /**
+   * SSL/TLS context to use instead of the system default. It must have been configured with {@link
+   * GrpcSslContexts}, but options could have been overridden.
+   *
+   * @param sslContext the ssl context
+   */
+  public void setSslContext(SslContext sslContext) {
```

Comment:
@xiang90 it seems so. if you look the grpc/auth guide for java; "ssl" is used instead of "tls" in naming. https://grpc.io/docs/guides/auth.html#with-server-authentication-ssltls-4